arviz/plots/backends/bokeh/forestplot.py

```diff
@@ -197,12 +197,13 @@ def plot_forest(
     if kind == "ridgeplot":  # space at the top
         y_max += ridgeplot_overlap
 
-    axes[0, 0].y_range._property_values[
-        "start"
-    ] = -all_plotters[  # pylint: disable=protected-access
-        0
-    ].group_offset
-    axes[0, 0].y_range._property_values["end"] = y_max  # pylint: disable=protected-access
+    for i in range(idx):
+        axes[0, i].y_range._property_values[
+            "start"
+        ] = -all_plotters[  # pylint: disable=protected-access
+            0
+        ].group_offset
+        axes[0, i].y_range._property_values["end"] = y_max  # pylint: disable=protected-access
```

Comment:
can this go above? I think the ymax definition does not depend on anything in between and can also be moved before the loop above. There already is a loop over axes, now that we loop here for the yaxis lims I think we can use the same loop that starts in line 175, mostly for clarity